src/runtime/link-accounts-flow-test.js

```diff
@@ -235,8 +236,8 @@ describes.realWin('LinkCompleteFlow', {}, (env) => {
 
     await startPromise;
     expect(startStub).to.be.calledWithExactly();
-    const activityIframeView = await instance.activityIframeViewPromise_;
```

Comment:
Would there be value in still awaiting this, to verify it doesn't throw?

Reply:
There's no more promise to await. This area only tests what happens in the `LinkCompleteFlow` constructor. Since it stubs the `start` function which is where I moved that logic to, this private field that the test expects to exist on the class now is just part of the promise chain in the `start` method so we can't even test the exact same assertion. That is also why I expanded the test below to check for different indexes and now this test only checks that the message passed in the `ActivityResult` is the same `response` that gets passed into `LinkCompleteFlow`